pyproject.toml

```diff
@@ -0,0 +1,94 @@
+[build-system]
+requires = ["setuptools", "lsst-versions >= 1.3.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lsst-multiprofit"
+description = "Astronomical image and source model fitting code."
+license = {file = "LICENSE"}
+readme = "README.rst"
+authors = [
+    {name="Rubin Observatory Data Management", email="dm-admin@lists.lsst.org"},
+]
+classifiers = [
```

Comment:
Add GPL license to the classifier list and maybe there is a Science/Research intended audience and topic Scientific/Engineering :: Astronomy (see astropy)?